torchci/pages/metrics.tsx

```diff
@@ -757,6 +757,11 @@ export default function Page() {
                 type: "string",
                 value: Intl.DateTimeFormat().resolvedOptions().timeZone,
               },
+              {
+                name: "repo",
+                type: "string",
+                value: "pytorch/%",
```

Comment:
Oh, I didn't know that % in a string param could be used in a LIKE statement. TIL